### PR TITLE
JWArdoin phase01_task03

### PR DESCRIPTION
**Pull request by jwardoin**

### CODE REVIEW

File: setup_wordpress.sh [hunk 1]
```diff
@@ -1,6 +1,111 @@
...
+sudo mysql -u root < /tmp/db.setup
+sudo rm /tmp/db.setup
+
+# Configure NGINX
+
+sudo touch wordpress.conf /etc/nginx/sites-available
+sudo cat >/etc/nginx/sites-available/wordpress.conf <<EOF
+server {
+	listen 80;
+	root $path;
```
Comment (korney4eg) [inline setup_wordpress.sh:37]: you have `path` here, but defined `Path` ;)

File: setup_wordpress.sh [hunk 2]
```diff
@@ -1,6 +1,112 @@
-#!/bin/bash
-# Please do not remove this line. This command tells bash to stop executing on first error. 
-set -e
+#!/bin/bash -xe
```
Comment (korney4eg) [inline setup_wordpress.sh:1]: you create VM using command `vagrant up`.

There is antoher command `vagrant provision` that allows you to rerun your scripts without recreating VM. Could you try to make it work?

**Comment by korney4eg** [comment]
hey @jwardoin were you able to look into my comment?

**Comment by jwardoin** [comment]
hey @korney4eg, sorry for the late response. I started building a cloud resume after passing my SAA exam and then got COVID. Just getting over it. I hope to finish my resume and find time to return to your tasks soon!

**Comment by korney4eg** [comment]
closing for now

**Pull request closed without merging**